### ISSUE TITLE
vscode-1.33.1: rejected patch 0003

### ISSUE THREAD

**Issue by eatradish** [AOSC-Dev/aosc-os-abbs#1836]
Bug description
-----------------------
my `autobuild/patch` file:

```
root@explosive-saki-40004 [ autobuild@(HEAD detached at 1.33.1) ] # cat patch
COMMIT=$(git rev-parse HEAD)
DATE=$(date -u '+%FT%TZ')
cat << 'EOF' | sed "s|@COMMIT@|${COMMIT}|g; s|@DATE@|${DATE}|g" > autobuild/patches/0003-product-json-add-extension-market-function.patch
diff --git a/product.json b/product.json
index bf75a611c5..d461f247b9 100644
--- a/product.json
+++ b/product.json
@@ -1,4 +1,17 @@
 {
+       "quality": "stable",
+       "extensionsGallery": {
+               "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
+               "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
+               "itemUrl": "https://marketplace.visualstudio.com/items"
+       },
+       "documentationUrl": "https://code.visualstudio.com/docs",
+       "requestFeatureUrl": "https://github.com/Microsoft/vscode/issues",
+       "introductoryVideosUrl": "https://go.microsoft.com/fwlink/?linkid=832146",
+       "tipsAndTricksUrl": "https://go.microsoft.com/fwlink/?linkid=852118",
+       "keyboardShortcutsUrlLinux": "https://go.microsoft.com/fwlink/?linkid=832144",
+       "commit": "@COMMIT@",
+       "date": "@DATE@",
        "nameShort": "Code - OSS",
        "nameLong": "Code - OSS",
        "applicationName": "code-oss",
-- 
2.21.0

EOF

for i in autobuild/patches/*; do
    abinfo "Applying ${i}..."
    patch -Np1 --binary -i "$i" || exit 1
done
```
why rejct...?

**Comment by eatradish** [comment]
and `product.json.rej`:

```
root@explosive-saki-40004 [ autobuild@(HEAD detached at 1.33.1) ] # cat ../product.json.rej 
--- product.json
+++ product.json
@@ -1,4 +1,17 @@
 {
+       "quality": "stable",
+       "extensionsGallery": {
+               "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
+               "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
+               "itemUrl": "https://marketplace.visualstudio.com/items"
+       },
+       "documentationUrl": "https://code.visualstudio.com/docs",
+       "requestFeatureUrl": "https://github.com/Microsoft/vscode/issues",
+       "introductoryVideosUrl": "https://go.microsoft.com/fwlink/?linkid=832146",
+       "tipsAndTricksUrl": "https://go.microsoft.com/fwlink/?linkid=852118",
+       "keyboardShortcutsUrlLinux": "https://go.microsoft.com/fwlink/?linkid=832144",
+       "commit": "51b0b28134d51361cf996d2f0a1c698247aeabd8",
+       "date": "2019-05-13T13:15:22Z",
        "nameShort": "Code - OSS",
        "nameLong": "Code - OSS",
        "applicationName": "code-oss",
```

**Comment by eatradish** [comment]
fix of b24a7675bcf700f7a5bca1065a17ad4541cfa770, close this issue
